### DISCUSSION TITLE
Add local variables to exception?

### DISCUSSION THREAD

**Discussion by jokaorgua** [getsentry/sentry-symfony#599] [Q&A] [answered]
Hello everyone. 

Could anyone explain how to add local variables to exceptions to be able to view them in sentry WebUI?

![image](https://user-images.githubusercontent.com/5228422/154432508-376eb847-cb5e-41fb-b75d-2a3de98abd2e.png)
on this image found on https://sentry.io/for/php/ we can see that it is possible, somehow, to capture local variables.

But I can not find this in code.

Thanks

**Comment by jokaorgua** [comment, marked as answer]
for those who have found this.

check the PHP in zend.exception_ignore_args value. it mas be set to Off

**Reply by stayallive** [reply]
Correct, see also our docs on this: https://docs.sentry.io/platforms/php/guides/symfony/troubleshooting/#missing-variables-in-stack-traces
